spec/computable/helpers.ts

```diff
@@ -26,27 +30,70 @@ function getDefaults(): any[] {
   ]
 }
 
-export async function deployToken(web3:any, account:string): Promise<Contract> {
-  const eip20 = await new web3.eth.Contract(tokenJson.abi, undefined, { gasPrice: GAS_PRICE, gas: GAS })
-    .deploy({ data: tokenJson.bytecode, arguments: [
-      Token.supply,
-      Token.name,
-      Token.decimals,
-      Token.symbol
-    ]})
+/**
```

Comment:
This is a neat bit of refactoring. A generic deploy function seems useful to have around.

Reply:
:fact